subsys/net/l2/openthread/Kconfig.thread

```diff
@@ -52,3 +52,13 @@ config OPENTHREAD_POLL_PERIOD
 	int "Poll period for sleepy end devices [ms]"
 	default 236000
 	depends on OPENTHREAD_MTD_SED
+
+config OPENTHREAD_MAX_CHILDREN
+	int "The maximum number of children"
+	range 10 512
```

Comment:
Not sure about adding a lower limit on the maximum number of children supported (is it specified in spec that router must support at least 10 children?).

Reply:
I think 10 is minimum to pass the certification.

Reply:
The requirement comes from the Thread Conformance document.